docs/api.rst

```diff
@@ -72,6 +72,7 @@ Kymotracking
     :toctree: _api
 
     kymotracker.kymoline.KymoLine
+    kymotracker.kymoline.KymoLine.estimate_diffusion
```

Comment:
I think this should already be generated from the line above

Reply:
I thought that as well, but if I omit it, it doesn't appear. I think it's different from the other objects since in the other cases the objects are explicitly imported in the init.

If I omit it, I get this
```
checking consistency... C:\Dev\Pylake\pylake\docs\_api\lumicks.pylake.kymotracker.kymoline.KymoLine.estimate_diffusion.rst: WARNING: document isn't included in any toctree
```
and the link goes dead.